core-bundle/src/EventListener/DataContainer/DefaultOperationsListener.php

```diff
@@ -221,6 +221,15 @@ private function isGranted(string $actionClass, string $table, DataContainerOper
         return $this->security->isGranted(ContaoCorePermissions::DC_PREFIX.$table, $subject);
     }
 
+    private function copyAction(string $table, DataContainerOperation $operation): CreateAction
+    {
+        $new = $operation->getRecord();
+        unset($new['id']);
+        $new['sorting'] = ($new['sorting'] ?? 0) + 1;
```

Comment:
```suggestion
        $new['sorting'] = ($new['sorting'] ?? 0) + 128;
```

At least that is how we do it in `DC_Table`.

https://github.com/contao/contao/blob/ab79c30add2d927691fa29914685501822bf137a/core-bundle/contao/drivers/DC_Table.php#L1313

Reply:
well not exactly. 128 is the default value, but something can be sorted in between after resorting. +1 is the best attempt at telling "this is exactly after this".